.github/workflows/release.yml

```diff
@@ -0,0 +1,90 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  ci:
+    name: Lint and Test
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+
+    - name: Restore npm cache
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+    
+    - name: Install npm dependencies
+      run: npm ci
+
+    - name: Lint
+      run: npm run lint
+
+    - name: Test
+      run: npm run test
+
+    - name: Upload Screenshots
+      if: failure()
+      uses: actions/upload-artifact@v2
+      with:
+        name: cypress-screenshots
+        path: cypress/screenshots
+
+    - name: Upload Video
+      uses: actions/upload-artifact@v2
+      if: always()
+      with:
+        name: cypress-videos
+        path: cypress/videos
+
+  build-frontend:
```

Comment:
Just preference, but I tend to prefer `build:frontend`, so that the colon clearly signifies a sub-task

Reply:
Unfortunately I can't do that. I tried a string but it wouldn't allow me 
![image](https://user-images.githubusercontent.com/44710980/96057501-b4e3d480-0e4e-11eb-945c-f924e0c25a61.png)

Reply:
try `build::frontend`

Reply:
nope 
![image](https://user-images.githubusercontent.com/44710980/97443854-81e31b80-18f9-11eb-83af-cf916aba55b6.png)

Reply:
Hmm, ok. I'm pretty sure I've used it before in GitLab yml